### DISCUSSION TITLE
HeatIT - Z-PH Wall Controller - KeyHeldDown immediately goes back to KeyReleased

### DISCUSSION THREAD

**Discussion by GD-Dal** [zwave-js/zwave-js#6706] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?
When longpressing the rocker on my Heatit Z-PH WALL CONTROLLER, it goes back to KeyReleased right away, and nothing more is happening

What did you expect to happen instead?

I expected to see the KeyReleased command when I released the rocker, and not before

  1. By longpressing any button on the wall controller

Anything else we should know?

### Software versions

Home Assistant Z-Wave Integration: 
Driver version: 12.4.4
Server version: 1.34.0
Connected via Z‐Stick Gen5 USB Controller

### Device information

Manufacturer: Heatit
Model name: Z-PH WALL CONTROLLER
Node ID: 36
[Information](https://media.heatit.com/2077)

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[config_entry-zwave_js-7a73b0eb0581b61be03ee1942ecc087a.json](https://github.com/zwave-js/node-zwave-js/files/14771607/config_entry-zwave_js-7a73b0eb0581b61be03ee1942ecc087a.json)
[home-assistant_zwave_js_2024-03-27T07-52-54.057Z.log](https://github.com/zwave-js/node-zwave-js/files/14771608/home-assistant_zwave_js_2024-03-27T07-52-54.057Z.log)

**Comment by AlCalzone** [comment, marked as answer]
According to your log that this is what the device sends. So you'll have to take this to HeatIt support.
